zenmap/share/zenmap/locale/es.po

```diff
@@ -0,0 +1,2597 @@
+"Translator: Marta GDLP <martagarciadelapaz@gmail.com>\n"
+"Last-Translator: Marta GDLP <martagarciadelapaz@gmail.com>\n"
+"Language-Team: Spanish\n"
+"Language: es\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: zenmapCore/NmapParser.py:380
+msgid "Unknown Host"
+msgstr "Host Desconocido"
+
+#: zenmapCore/NmapParser.py:421
+msgid "unknown"
+msgstr "desconocido"
+
+#: zenmapCore/NmapParser.py:424
+msgid "Unknown version"
+msgstr "Versión desconocida"
+
+#: zenmapCore/NmapParser.py:427 zenmapGUI/ScanOpenPortsPage.py:445
+#: zenmapGUI/ScanOpenPortsPage.py:459 zenmapGUI/SearchGUI.py:534
+msgid "Unknown"
+msgstr "Desconocido"
+
+#: zenmapCore/NmapParser.py:713
+#, python-format
+msgid "%s on %s"
+msgstr "%s en %s"
+
+#: zenmapCore/UmitDB.py:139
+msgid "No module named dbapi2.pysqlite2 or sqlite3"
+msgstr "Ningún módulo nombrado dbapi2.pysqlite2 o sqlite3 "
+
+#: zenmapCore/UmitOptionParser.py:140
+#, python-format
+msgid "Use DIR as the user configuration directory. Default: %default"
+msgstr "Utiliza DIR como directorio de configuración de usuario. Por defecto: "
+"%default"
+
+#: zenmapCore/UmitOptionParser.py:154
+msgid ""
+"Specify a scan result file in Nmap XML output format. Can be used more than "
+"once to specify several scan result files."
+msgstr ""
+"Especificar un archivo de resultados de escaneo en el formato XML de Nmap. Puede "
+"ser utilizado más de una vez para especificar varios archivos de resultados "
+"de escaneo."
+
+#: zenmapCore/UmitOptionParser.py:165
+#, python-format
+msgid "Run %s with the specified args."
+msgstr "Ejecutar %s con los argumentos especificados."
+
+#: zenmapCore/UmitOptionParser.py:173
+msgid ""
+"Begin with the specified profile selected. If combined with the -t (--"
+"target) option, automatically run the profile against the specified target."
+msgstr ""
+"Empieza con el perfil específico seleccionado. Si está combinado con la opción -t "
+"(--target), ejecuta automáticamente el perfil contra el objetivo "
+"especificado."
+
+#: zenmapCore/UmitOptionParser.py:184
+msgid ""
+"Specify a target to be used along with other options. If specified alone, "
+"open with the target field filled with the specified target"
+msgstr ""
+"Especifica un objetivo para ser usado junto a otras opciones. Si es especificado "
+"solo, abrir con el campo objetivo rellenado con el objetivo especificado"
+
+#: zenmapCore/UmitOptionParser.py:192
+msgid ""
+"Increase verbosity of the output. May be used more than once to get even "
+"more verbosity"
+msgstr ""
+"Aumenta la verbosidad de la salida. Puede ser utilizado más de una vez para "
+"obtener aún más verbosidad"
+
+#: zenmapCore/NmapCommand.py:139
+msgid "Python 2.4 or later is required."
+msgstr "Es necesario Python 2.4 o una versión más reciente."
+
+#: zenmapGUI/ScriptInterface.py:262
+msgid ""
+"There was an error getting the list of scripts from Nmap. Try upgrading Nmap."
+msgstr ""
+"Ha habido un error obteniendo la lista de scripts de Nmap. Intente "
```

Comment:
I noticed the same thing. I did the translation as it was in the english version but we should stick to the formal one.